File: code/TemplateStudioForUWP/Templates/Uwp/_comp/MVVMToolkit/Service.IdentityCommon._VB/ViewModels/UserViewModel.vb
Imports Param_RootNamespace.Helpers
Imports Windows.UI.Xaml.Media.Imaging
Imports Microsoft.Toolkit.Mvvm.ComponentModel

Namespace ViewModels
    Public Class UserViewModel
        Inherits ObservableObject

        Private _name As String
        Private _userPrincipalName As String
        Private _photo As BitmapImage

        Public Property Name As String
            Get
                Return _name
            End Get
            Set(value As String)
                SetProperty(_name, value)
            End Set
        End Property

        Public Property UserPrincipalName As String
            Get
                Return _userPrincipalName
            End Get
            Set(value As String)
                SetProperty(_userPrincipalName, value)
            End Set
        End Property

        Public Property Photo As BitmapImage
            Get
                Return _photo
            End Get
            Set(value As BitmapImage)
                SetProperty(_photo, value)
            End Set
        End Property

        Public Sub New()
        End Sub
    End Class
End Namespace

File: code/TemplateStudioForUWP/Templates/_comp/MT/IdentityCommon/ViewModels/UserViewModel.cs
using System;
using Param_RootNamespace.Helpers;
using Windows.UI.Xaml.Media.Imaging;
using Microsoft.Toolkit.Mvvm.ComponentModel;

namespace Param_RootNamespace.ViewModels
{
    public class UserViewModel : ObservableObject
    {
        private string _name;
        private string _userPrincipalName;
        private BitmapImage _photo;

        public string Name
        {
            get => _name;
            set => SetProperty(ref _name, value);
        }

        public string UserPrincipalName
        {
            get => _userPrincipalName;
            set => SetProperty(ref _userPrincipalName, value);
        }

        public BitmapImage Photo
        {
            get => _photo;
            set => SetProperty(ref _photo, value);
        }

        public UserViewModel()
        {
        }
    }
}
